packages/xo-web/src/common/intl/messages.js

```diff
@@ -1134,6 +1134,11 @@ const messages = {
   vmRevertSuccessfulTitle: 'Revert successful',
   vmRevertSuccessfulMessage: 'VM successfully reverted',
 
+  // ----- VM backup tab -----
+  goToBackupPage: 'Go to the backup page.',
+  vmInLegacyBackup:
+    'This VM is backed up using the legacy backup system. Go to see the legacy backups.',
```

Comment:
We don't know that, we're not checking. All we know is that the user *has* some legacy backup jobs:
`'This VM may be backed up by the legacy backup system. See legacy jobs.'`